test/system/projects_test.rb

```diff
@@ -70,6 +70,41 @@ class ProjectsTest < ApplicationSystemTestCase
 
     assert_text("Sorry, you don't have access to that")
   end
+  
+  test 'iteration card displays when there are no updates' do
+    create_project
+    @project = Project.last
+    iteration = create(:committed_iteration, project: @project)
+    visit project_path(@project)
+    assert_text('No updates')
+  end
+  
+  test 'iteration card displays latest check-in' do
+    create_project
+    @project = Project.last
+    iteration = create(:committed_iteration, project: @project)
+    @check_in = create(:check_in, iteration: iteration)
+    visit project_path(@project)
+    assert_text('Checked in on '+friendly_date(@check_in.complete_at))
```

Comment:
We probably need to hide or display a more general message (e.g. 3 check-in's completed) to stakeholders.